charts/node-exporter/values.yaml

```diff
@@ -0,0 +1,15 @@
+prometheus-node-exporter:
+  service:
+    type: ClusterIP
+    port: 9100
+    targetPort: 9100
+    # By default, the prometheus annotation is on the service,
+    # but I do not think this is the intended configuration.
+    # https://github.com/helm/charts/issues/20736
+    annotations: {}
+  podAnnotations:
+    prometheus.io/scrape: "true"
+    prometheus.io/port: "9100"
```

Comment:
@jpweber will you review this comment in particular? I wonder if you have had this experience in the past

Reply:
Short answer is both will work. But really it all depends on how your scrape config is setup to scrape the information. The default configuration with the scrape annotation is assuming you are using the `endpoints` role. If an end point is attached to a service all the service `__meta_` labels are included.

Reply:
ok I see https://prometheus.io/docs/prometheus/latest/configuration/configuration/#endpoints